simpleLooping/getSmallestLargest.py

```diff
@@ -3,3 +3,21 @@
 # Ex:
 #   countPositiveAndNegativeInteger([-9, -5, -2, -6, -3, -1]) # return (-9, -1)
 #   countPositiveAndNegativeInteger([3, 5, 7, -2, 6, -1, -3]) # return (-3, 7)
+from typing import List, Tuple
+
+def getSmallestLargest(lst: List[int]) -> Tuple[int, int]:
+    lenLst = len(lst)
+
+    smallestNumber = lst[0]
+    largestNumber = lst[0]
+    i = 0
```

Comment:
Should you start with index 1? Because if not, you will compare the value of lst[0] with lst[0] themself.

I agree this might be tiny. But for example we only process a list of 3 to 5 numbers at a time, this tiny thing will cost 25% to 30% addition runtime, which can be a hugh impact in large scale operatin.

In additional, I see you do a great job in the previous 2 problems where you set i = 1, so this is just a bug where we did not carefully evaluate the token “0” here. Remember, “why we have this token here?” Is a really important question.

Reply:
acknowledge